notes_app/public/index.css

```diff
@@ -186,26 +162,117 @@ button:focus {
 }
 
 .gg-trash::after {
-    /* positioning */
     top: -4px;
     left: -5px;
-    /* display & box model */
     width: 16px;
     height: 2px;
     border-radius: 3px;
-    /* color */
     background: currentColor;
 }
 
 .gg-trash::before {
-    /* positioning */
     top: -7px;
     left: -2px;
-    /* display & box model */
     width: 10px;
     height: 4px;
     border: 2px solid;
     border-bottom: transparent;
     border-top-left-radius: 2px;
     border-top-right-radius: 2px;
 }
+
+/* Split Pane Gutter Styles */
+.gutter {
+    background-color: #eee;
+    background-repeat: no-repeat;
+    background-position: 50%;
+}
+
+.gutter.gutter-horizontal {
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
+}
+
+.gutter.gutter-horizontal:hover {
+    cursor: col-resize;
+}
+
+/* Auth Container Wrapper */
+.auth-container {
+    padding: 2rem;
+    text-align: center;
+}
+
+/* Login & Signup Form Styles */
+.login-wrapper {
```

Comment:
Please split this file and move to a new file: login.css CCS classes used for the login page.

Also, it is a good habbit, that every css class in on a section starts from the same prefix, please use "login"